setup.py

```diff
@@ -6,7 +6,7 @@
 
 # aiohttp requirement is pegged as we need to manually ensure that
 # https://github.com/aio-libs/aiobotocore/pull/248 will continue working
-install_requires = ['botocore>=1.5.71, <=1.5.78', 'aiohttp>=2.0.4, <=2.3.0',
+install_requires = ['botocore>=1.5.71, <=1.5.78', 'aiohttp>=2.X.X, <=2.X.X',
```

Comment:
updates her should fix CI

Reply:
ya, broken on purpose for now so we set the right version that's needed

Reply:
got it